src/nxdoc/web-ui/web-ui-customization/web-ui-managing-translations.md

```diff
@@ -57,6 +57,65 @@ As an example, take the following three labels which are used in the `nuxeo-acti
 ```
 {{/panel}}
 
+In order to make your bundle append additional label translations, you need to create the proper `messages` files as follow:
+
+```.
+├── pom.xml
+├── src
+│   └── main
+│       └── resources
+│           ├── META-INF
+│           │   └── MANIFEST.MF
+│           ├── OSGI-INF
+│           │   ├── deployment-fragment.xml
+│           │   ...
+│           └── web
+│               └── nuxeo.war
+│                   └── ui
+│                       ├── i18n
+│                       │   ├── messages-fr.json
+│                       │   ├── messages.json
+│                       │   └── messages-us.json
```

Comment:
ok but `messages-us.json`still doesn't exist in `nuxeo-web-ui`, and it causes an error